### ISSUE TITLE
lb4 command hangs

### ISSUE THREAD

**Issue by sagark1510** [strongloop/loopback-next#3187]
## it should create necessary files and project get created

node: 10.16
os: macOS
npm: 6.9.0

I don't understand why it hangs.

**Comment by dougal83** [comment]
@sagark1510 Hey, might be helpful to post installed loopback version (`lb4 -v`) and any error messages.

**Comment by kerolloz** [comment]
## Description / Steps to reproduce / Feature proposal

I just write 
```shell
lb4 app
```
in my terminal.  

## Current Behavior

After entering the project name and description. My terminal stops(hangs), the cursor keeps blinking, and doesn't accept any input at `? Project root directory` question  pressing `<Enter>`, `Ctrl+Z`, `Ctrl+C` or `any button` has no effect.

![screenshot](https://user-images.githubusercontent.com/36763164/59862180-99fc6f80-9382-11e9-80e4-c88a5e6d40e1.png)


## Expected Behavior

I should be able to enter the project root directory.

## Note

I have just installed loopback cli, using the following command.

```shell
sudo npm i -g @loopback/cli
```
I use Ubuntu 19.04.

**Comment by dougal83** [comment]
@kerolloz Hey, I've just updated and can confirm the bug.  Posting version info:

````@loopback/* dependencies:
  - @loopback/authentication: ^2.1.2
  - @loopback/boot: ^1.4.1
  - @loopback/build: ^2.0.0
  - @loopback/context: ^1.19.1
  - @loopback/core: ^1.8.2
  - @loopback/metadata: ^1.2.2
  - @loopback/openapi-spec-builder: ^1.2.2
  - @loopback/openapi-v3-types: ^1.1.2
  - @loopback/openapi-v3: ^1.6.2
  - @loopback/repository-json-schema: ^1.7.0
  - @loopback/repository: ^1.7.0
  - @loopback/rest: ^1.16.0
  - @loopback/testlab: ^1.6.0
  - @loopback/docs: ^1.23.0
  - @loopback/example-hello-world: ^1.2.2
  - @loopback/example-log-extension: ^1.2.2
  - @loopback/example-rpc-server: ^1.2.2
  - @loopback/example-todo: ^1.6.2
  - @loopback/example-soap-calculator: ^1.6.3
  - @loopback/service-proxy: ^1.2.2
  - @loopback/http-caching-proxy: ^1.1.2
  - @loopback/http-server: ^1.4.1
  - @loopback/example-todo-list: ^1.7.2
  - @loopback/dist-util: ^0.4.0
  - @loopback/rest-explorer: ^1.2.2
  - @loopback/eslint-config: ^1.1.2
  - @loopback/example-express-composition: ^1.4.2
  - @loopback/example-greeter-extension: ^1.3.2
  - @loopback/booter-lb3app: ^1.2.2
  - @loopback/example-lb3-application: ^1.1.2
  - @loopback/example-greeting-app: ^1.1.2
  - @loopback/example-context: ^1.2.2````

**Comment by sagark1510** [comment]
@dougal83  glad you found the bug. @loopback/cli version: 1.17.0 this is the version I am using and there is not error or anything just it hangs.

**Comment by raymondfeng** [comment]
What's the node.js version?

**Comment by sagark1510** [comment]
@raymondfeng `10.16.0`

**Comment by raymondfeng** [comment]
Hmm, I can reproduce it with Node 12.3.1 on Mac too. :-(

**Comment by dougal83** [comment]
I just updated node to 12.4 on Win 10.  Still occuring.
Is it an issue with generators as a whole? https://github.com/yeoman/yeoman/issues/1714

**Comment by raymondfeng** [comment]
It might be caused by `inquirer@6.4.0`.

**Comment by raymondfeng** [comment]
See https://github.com/SBoudrias/Inquirer.js/issues/807

**Comment by raymondfeng** [comment]
Duplicate of https://github.com/SBoudrias/Inquirer.js/issues/811

**Comment by sagark1510** [comment]
@raymondfeng how do we do that? can we release new version with `inquirer@6.3.1` ??

**Comment by raymondfeng** [comment]
I'll wait a bit to see if the upstream bug can be fixed quickly. We can try to ping it to `inquirer@6.3.1` if necessary tomorrow.

**Comment by raymondfeng** [comment]
Released @loopback/cli@1.17.1
